Create a partial content for EU configuration in Studio (#158)
mulesoft/docs-studio: modules/ROOT/pages/_partials/eu-cloud-access.adoc

> = To Access the EU Control Plane (Anypoint Studio)

Reviewer: ```suggestion
= Access the EU Control Plane from Anypoint Studio
```

> You can configure your Anypoint Studio 6.4.2 and later to point to the EU control plane.
> 
> This process enables you to perform tasks such as deploying to CloudHub, importing an asset from Exchange, or importing an API definition from Design Center, in the EU control plane.

Reviewer: Suggest combining ability with reason why, something like...:
```suggestion
Configure your Anypoint Studio 6.4.2 and later implementation to point to the EU control plane to perform tasks there, such as deploying to CloudHub, importing an asset from Exchange, or importing an API definition from Design Center.
```

> . Open the preferences view. +

Reviewer: ```suggestion
. Open the preferences view in the manner appropriate to your OS: +
```

> * In MacOS navigate to *Anypoint Studio* > *Preferences..* or hit `cmd + ,`.

Reviewer: ```suggestion
* In MacOS, either navigate to *Anypoint Studio* > *Preferences...* or press `cmd + ,`.
```

"Preferences has just two dots .. or three ...?

GLOBALLY replace "hit" with "press", and clarify exactly which keys the user is pressing. These are currently formatted as syntax parameters, but they should be represented as UI controls/keycaps: "or press the *control* key simultaneously with the *,* key"?

* In Windows and Linux, navigate to *Window* > *Preferences...*.

> . Expand *Anypoint Studio* in the left navigation bar, and select *Anypoint Platform*.

Reviewer: ```suggestion
. Expand *Anypoint Studio* in the left navigation bar, and select *Anypoint Platform*:
```
GLOBALLY end text that directly introduces a list or an image (or a table or an example) with a colon.

+
image::studio::studio-anypoint-platform-settings.png[]

> . Select *EU* option from the region dropdown menu, and then select *Apply and Close*.

Reviewer: ```suggestion
. Select *EU* from the region drop-down menu, and then select *Apply and Close*:
```

+
image::studio::select-eu-region.png[]

> All Anypoint Platform tools embedded in Anypoint Studio will now display your Anypoint Platform from the EU Control plane.

Reviewer: ```suggestion
As a result, all Anypoint Platform tools embedded in Anypoint Studio display your Anypoint Platform from the EU control plane.
```
